.gitignore

```diff
@@ -2,3 +2,6 @@
 **/__pycache__/
 **/node_modules/
 .DS_Store
+app/.venv/
+.gitignore
+app/openssl.cnf
```

Comment:
Why ignoring the `openssl.cnf` file?
It's still required (until https://github.com/fmartinou/tydom2mqtt/pull/98 removes it)

Reply:
just a mistake, sorry